php/routes.php

```diff
@@ -26,4 +26,42 @@
             ]
         );
     });
+
+    CBPRouter::code('/code', function ($payload) {
+        $parsedUrl = wp_parse_url($payload['url']);
+
+        $isValidHost = $parsedUrl['host'] === 'gist.githubusercontent.com' ||
+            $parsedUrl['host'] === 'raw.githubusercontent.com' ||
+            $parsedUrl['host'] === 'github.com';
```

Comment:
Maybe make a list and check the list, like:

```php
$validHosts = ['gist.githubusercontent.com', 'raw.githubusercontent.com', 'github.com']; 

if (!in_array($parsedUrl['host'], $validHosts)) {

Reply:
Ah, good point. Fixed it in [this commit](https://github.com/KevinBatdorf/code-block-pro/pull/252/commits/d1564e6d91f540b7117cc6601431ef64611daa45)